docs/design/unified-image-specification.md

```diff
@@ -0,0 +1,108 @@
+
+# Unified image specification for node pools
+
+## Motivation
+
+There are inconsistencies in where and how you can specify images in the aks-engine cluster definitions.
+This leads to unnecessary complexity in the aks-engine codebase and potential confusion for aks-engine users.
+
+Additionally, it is not possible to specify different images for different pools running Windows which breaks some upgrade/test scenarios.
+
+Images for virtual machines in Azure are specified the same way regardless of guest OS. This proposal aims to unify how aks-engine cluster defintions specify images for Linux and Windows.
+
+## Current behavior
+
+### Linux
+
+Images are specified in the agentPoolProfile one of the following ways. 
+Specifing the image in agentPoolProfile allows multiple pools running different images.
+
+* Distro - Specifies a curated list of images for various 'distros' (pulled from marketplace or shared image gallery)
```

Comment:
the distros specify the Linux distribution to use and are mapped to an image from the marketplace  (not a SIG)